ramls/item.json

```diff
@@ -296,6 +296,59 @@
       "folio:linkFromField": "holdingsRecordId",
       "folio:linkToField": "id",
       "folio:includedElement": "holdingsRecords.0"
+    },
+    "lastCheckIn": {
+      "type": "object",
+      "readonly": true,
+      "additionalProperties": false,
+      "description": "Information about when an item was last scanned on the Inventory app.",
+      "properties": {
+        "dateTime": {
+          "type": "string",
+          "description": "Date and time of the last check in of the item.",
+          "format": "date-time"
+        },
+        "servicePointInfo": {
+          "type": "object",
+          "additionalProperties": false,
+          "description": "Service point being used by a staff member when item was scanned in Check in app.",
+          "properties": {
+            "id": {
+              "type": "string",
+              "description": "Service point ID being used by a staff member when item was scanned in Check in app.",
+              "pattern": "[A-F0-9]{8}-[A-F0-9]{4}-4[A-F0-9]{3}-[89AB][A-F0-9]{3}-[A-F0-9]{12}"
+            },
+            "name": {
+              "type": "string",
+              "description": "Service point name being used by a staff member when item was scanned in Check in app."
+            }
+          }
+        },
+        "source": {
+          "type": "object",
+          "additionalProperties": false,
+          "description": "Last name, first name middle name of a staff member who scanned the item",
+          "properties": {
```

Comment:
Hi @marcjohnson-kint, @mrdpopov, do we really need first/last/middle names to be set along with `staffMemberId`? 

I think, for similar cases we retrieving this additional info in business modules. For example `item.effective/permanent/tempLocation`s - we store only Id but then mod-inventory adds `name` attribute.

Reply:
Id is for the link to the user, other fields reduce the traffic in clients of this API.

Reply:
Ok. Does it make sense then to specify `staffMemberId` and `first/lastName` to be required fields? So there will be no situation when `staffMemberId` and names are out of sync (i.e. a client has updated id but names are still referencing the old user).